FAQ.md

```diff
@@ -34,6 +34,7 @@ Summarize the question and quote the reply, linking back to the original comment
 * [What semver version should I use?](#what-semver-version-should-i-use)
 * [Is it OK to make backwards-incompatible changes now?](#is-it-ok-to-make-backwards-incompatible-changes-now)
 * [My dependers don't use `dep` yet. What should I do?](#my-dependers-dont-use-dep-yet-what-should-i-do)
+* [One or more of my dependencies doesn't tag its releases. What should I do?](#one-or-more-of-my-dependencies-dont-tag-their-releases-what-should-i-do)
```

Comment:
Let's simplify the question to "How do I configure a dependency that doesn't tag its releases?"